src/Context/HTMLContext.php

```diff
@@ -15,6 +16,7 @@ class HTMLContext extends BaseContext
         'https://validator.nu/',
         'https://validator.w3.org/nu/',
     ];
+    private const HTML5_DOCTYPE_MARKUP = '<!doctype html>';
```

Comment:
While using ``assertInverse`` in ``thePageHtml5DoctypeDeclarationShouldNotBeValid``, we have reduced the ``HTML5_DOCTYPE_MARKUP`` usage to only one time, now we can safely move it to ``thePageHtml5DoctypeDeclarationShouldBeValid``.
```suggestion
```